libfreerdp/core/transport.c

```diff
@@ -374,6 +375,59 @@ BOOL transport_connect_nla(rdpTransport* transport)
 	return rdp_client_transition_to_state(rdp, CONNECTION_STATE_NLA);
 }
 
+BOOL transport_connect_nla_ext(rdpTransport* transport)
```

Comment:
Hum this function is a copy'n paste of `transport_connect_nla`, instead I'd suggest to have a `transport_connect_nla_ex(rdpTransport* transport, BOOL isExt)` and `transport_connect_nla_ext` and `transport_connect_nla` would call this one setting the correct value for `isExt`.

Reply:
I agree, I remove the nla_ext function and add a BOOL param to connect_nla should be better